Core/src/org/sleuthkit/autopsy/centralrepository/contentviewer/DataContentViewerOtherCases.java

```diff
@@ -539,9 +540,21 @@ public boolean isSupported(Node node) {
         //  Is supported if this node
         //      has correlatable content (File, BlackboardArtifact) OR
         //      other common files across datasources.
-        return this.file != null
+        
+        EamDb eamDb = null;
+        try {
+            eamDb = EamDb.getInstance();
```

Comment:
Is there a reason not to use EamDb.isEnabled() here?

Reply:
Nah - I wasn't thinking.